README.md

```diff
@@ -1,8 +1,9 @@
-# Occson::Rails
-Short description and motivation.
+# OCCSON
+
+Store, manage and deploy Rails configuration securely with Occson.
 
 ## Usage
-How to use my plugin.
+[PLACEHOLDER FOR VIDEO]
```

Comment:
Hmm, one more thing - we can add the video _in addition_ to usage instructions (especially as the planned ones are focused on introducing the entire platform first). If I'm using this in a professional setting I don't want to go to youtube for implementation instructions, but read them. WDYT?